cmd/ddev/cmd/composer-create.go

```diff
@@ -22,11 +22,9 @@ var composerCreateYesFlag bool
 
 // ComposerCreateCmd handles ddev composer create
 var ComposerCreateCmd = &cobra.Command{
-	Use: "create [args] [flags]",
-	FParseErrWhitelist: cobra.FParseErrWhitelist{
-		UnknownFlags: true,
-	},
-	Short: "Executes 'composer create-project' within the web container with the arguments and flags provided",
+	DisableFlagParsing: true,
```

Comment:
I'm 90% sure `DisableFlagParsing` and `FParseErrWhitelist: cobra.FParseErrWhitelist{UnknownFlags: true}` are functionally equivalent as far as running the command goes - but `DisableFlagParsing` is explicitly required for the completion function to be called. Not sure why, couldn't find any docs on it, but that's why I've made this particular change.